src/NexusMods.Telemetry/EventSender.cs

```diff
@@ -200,14 +203,22 @@ void AppendBytes(byte[] data)
         }
     }
 
-    private static async ValueTask SendEvents(ReadOnlyMemory<byte> data, HttpClient httpClient, CancellationToken cancellationToken)
+    private async ValueTask SendEvents(ReadOnlyMemory<byte> data, HttpClient httpClient, CancellationToken cancellationToken)
     {
         using var request = new HttpRequestMessage(HttpMethod.Post, Constants.MatomoTrackingEndpoint);
         request.Content = new ReadOnlyMemoryContent(data);
         request.Content.Headers.ContentType = new MediaTypeHeaderValue(MediaTypeNames.Application.Json);
 
-        var response = await httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
-        response.EnsureSuccessStatusCode();
+        try
+        {
+            var response = await httpClient.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
+            response.EnsureSuccessStatusCode();
+        }
+        catch (Exception ex)
+        {
+            // NOTE (halgari): Yes, this will add more bloat to the log, but only in testing where DEBUG logging is enabled.
+            _logger.LogDebug(ex, "Error occured while sending events");
+        }
```

Comment:
We were talking about adding a circuit breaker with Polly to avoid the spam attempts. 
I guess this works fine for now. 

I would consider using warning, but if we don't do any circuit breaking this could get spammy, so I guess debug is fine if we go with this.

Reply:
Took a look at polly and I feel like it's a tad overkill in this case. The circuit breaker is great, but it has settings like "half-open", "minimum throughput" and "failure ratio" that don't really make sense on something that's already rate limited like telemetry. We don't really care about retries either as we'll just try again on the next polling event.